File: output/qwen_2.5_72b_instruct/avatar/direct_translation/java/python/atcoder_ABC171_B.py

import sys

def main():
    import sys
    input = sys.stdin.read
    data = input().split()
    n = int(data[0])
    k = int(data[1])
    a = list(map(int, data[2:]))
    a.sort()
    ans = sum(a[:k])
    print(ans)

if __name__ == "__main__":
    main()

File: output/gpt_4o/avatar/algo_based_translation/python/java/atcoder_ABC171_B.java

import java.util.Arrays;
import java.util.Scanner;

public class atcoder_ABC171_B {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        // Step 3: Input Handling
        int n = scanner.nextInt();
        int k = scanner.nextInt();

        // Step 4: Declare variable
        int[] numbers = new int[n];

        // Step 5: Input Handling
        for (int i = 0; i < n; i++) {
            numbers[i] = scanner.nextInt();
        }
        
        // Step 6: Data Processing Logic
        Arrays.sort(numbers);

        // Step 8: Data Processing Logic
        int sum = 0;
        for (int i = 0; i < k; i++) {
            sum += numbers[i];
        }

        // Step 9: Output Handling
        System.out.println(sum);

        scanner.close();
    }
}

